README.md

```diff
@@ -1,16 +1,120 @@
-# Title
+# String Repetition & Searching in JavaScript
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+* Demonstrate string repitition in JavaScript
+* Demonstrate searching in JavaScript
+* Use `repeat()` to duplicate strings
+* Declare a function that utilizes `search()` method
 
 ## Introduction
 
-## SWBAT 1
+Working with `String` values is one of the most popular functions
+in JavaScript. Strings are objects within the JavaScript language.
+Strings are not treated the same as arrays, so built-in functions
+must be used to work with their values. We have various functions
+that provide ways to access or manipulate the contents of a string
+such as with repitition and searching.
```

Comment:
This is subjective, but I feel like this paragraph could be rearranged to flow more easily. Maybe something like: "Strings are objects in the JavaScript language, and working with their values is one of the most common tasks a JavaScript developer performs."